bindings/rust/build.rs

```diff
@@ -5,7 +5,9 @@ fn main() {
     c_config
         .flag_if_supported("-Wno-unused-parameter")
         .flag_if_supported("-Wno-unused-but-set-variable")
-        .flag_if_supported("-Wno-trigraphs");
+        .flag_if_supported("-Wno-trigraphs")
+        .flag_if_supported("-Werror")
+        .flag_if_supported("-W!all"); // -Wextra -Wpedantic -Wvla -Wconversion
```

Comment:
```suggestion
        .flag_if_supported("-W!all"); // -Wextra -Wpedantic -Wvla -Wconversion
```

This was me testing if this is where the flags are passed to clang. It didn't work, this line can be removed

Reply:
Addressed in 0002b5f09453f48b80405020fb76db3d6c9088e3